code/frameworks/sveltekit/template/stories_svelte-kit-prerelease-ts/ButtonTypeScriptRunes1.svelte

```diff
@@ -0,0 +1,55 @@
+<script lang="ts">
```

Comment:
I think we can make these two files - `Runes1` and `Runes2` more testing focused. 

Instead of using the Button analogy, just have a Svelte component that takes in all types of props imaginable. strings, numbers, undefined, unions, snippets, imported types, etc. and then it just renders all the props in a list or something.
It doesn't matter what it renders or if it's pretty or not, all that we want to see is that the docgen works correctly in all cases. Eg. see this similar story demonstrating `react-docgen`: https://next--630511d655df72125520f051.chromatic.com/?path=/story/stories-renderers-react-ts-argtypes--ts-function-component

We can't actually render the ArgsTable because we don't have React, but we could perhaps render the `argTypes`, so we have regression testing via Chromatic. See https://github.com/storybookjs/storybook/blob/next/code/renderers/react/template/stories/ts-argtypes.stories.tsx/#L35 for inspiration.

And then do one in TS and one in JS with JSDoc.